sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSessionManager.java

```diff
@@ -47,43 +47,58 @@
 /**
  * Package-private class that manages session aware message receiving.
  */
-class UnnamedSessionManager implements AutoCloseable {
+class ServiceBusSessionManager implements AutoCloseable {
```

Comment:
Why the rename?

Reply:
Because now we are using this class for named session also.